package.json

```diff
@@ -20,6 +20,8 @@
     "test-cov-html": "lab -r html -o coverage.html",
     "test:cypress": "node_modules/.bin/cypress open",
     "test:cypress-silently": "npm run pretest && node_modules/.bin/cypress run && npm run test:cypress-prepare-report",
+    "test:cypress-edge": "npm run pretest && node_modules/.bin/cypress run --browser edge && npm run test:cypress-prepare-report",
```

Comment:
Please add Chrome Browser too. Cypress defaults to electron browser in the headless mode. Hence we need Chrome too.